### ISSUE TITLE
FrankaCubeStack not converging

### ISSUE THREAD

**Issue by MohammadYasar** [NVIDIA-Omniverse/IsaacGymEnvs#73]
Hi, I have been training FrankaCubeStack with the default parameters in cfg/task/FrankaCubeStack.yaml and cfg/train/FrankaCubeStackPPO.yaml. The reward stops increasing after approx. 800 and 10,000 epochs. I have also changed the parameter values for minibatch_size to 131072 and numEnvs to 16384 and kl_threshold to 1.6e-2, as reported in the paper, but haven't been able to get the results as reported in the paper. 
Here's the train command:
`python train.py headless=True task=FrankaCubeStack wandb_activate=True wandb_entity=nvidia wandb_project=FrankaCubeStack`

![image](https://user-images.githubusercontent.com/9808077/184187971-114ef372-856a-4e56-8d87-b27fb3601600.png)

Please let me know if there's something I'm doing wrong.

**Comment by DanielTakeshi** [comment]
I can reproduce the result from @MohammadYasar . Here is the curve I see on wandb:

![Screenshot from 2022-09-09 21-21-42](https://user-images.githubusercontent.com/5532096/189463348-d99be472-1847-452e-b23c-ea36c9c349d1.png)

I ran this using:

```
python train.py task=FrankaCubeStack headless=True wandb_activate=True wandb_entity=A wandb_project=B
```

where I replaced `A` with my wandb username and `B` with the project name I used. 

This is with the main branch and the latest commit as of today: https://github.com/NVIDIA-Omniverse/IsaacGymEnvs/commit/ca7a4fb762f9581e39cc2aab644f18a83d6ab0ba

I did not change any of the PPO settings.

Also, it took about 264 minutes to run which is a bit longer than expected; for reference, running the above but with `FrankaCabinet` took about 20 minutes.

**Comment by DanielTakeshi** [comment]
Quick follow-up, I decided to load the policy rollouts and visualize them:

https://youtu.be/ACwcYy3Xb08

From the video it actually seems like the Franka is learning to do Cube stacking? I think it's succeeding most of the time.

So maybe the reward scale is different now as compared to the Isaac Gym white paper as reported in https://arxiv.org/abs/2108.10470 ?

**Comment by MohammadYasar** [comment]
I think it should stack and move away from the cube. I was able to reproduce the graph by removing the `stack_reward>0 ` from Line 746:
 `reset_buf = torch.where((progress_buf >= max_episode_length - 1) , torch.ones_like(reset_buf), reset_buf)
` 
![W B Chart 9_30_2022, 5_48_34 PM](https://user-images.githubusercontent.com/9808077/193360902-41e5f2a3-1717-4796-98a9-48b3652a7a46.png)
